src/main/kotlin/com/fasterxml/jackson/module/kotlin/KotlinModule.kt

```diff
@@ -26,14 +26,18 @@ fun Class<*>.isKotlinClass(): Boolean {
  *                                      the default, collections which are typed to disallow null members
  *                                      (e.g. List<String>) may contain null values after deserialization.  Enabling it
  *                                      protects against this but has significant performance impact.
+ * @param   experimentalDeserializationBackend
+ *                                  Default: false.  Whether to enable experimental deserialization backend.  Enabling
+ *                                      it significantly improve performance in certain use cases.
  */
 class KotlinModule constructor (
     val reflectionCacheSize: Int = 512,
     val nullToEmptyCollection: Boolean = false,
     val nullToEmptyMap: Boolean = false,
     val nullIsSameAsDefault: Boolean = false,
     val singletonSupport: SingletonSupport = DISABLED,
-    val strictNullChecks: Boolean = false
+    val strictNullChecks: Boolean = false,
+    val experimentalDeserializationBackend: Boolean = false
```

Comment:
The bug of not passing the `experimentalDeserializationBackend` parameter from the `builder` has been fixed in the official branch, and tests have been added.

644b139e4f6c82678eaadf162838ac0b7be9de8b
1881b4aec3dffd1d762bca6c950aa70f08dff8fb

**Problem Location**
https://github.com/FasterXML/jackson-module-kotlin/pull/439/files#diff-9c01821bc332a383a4e1ac69dd85b9514e64d7bf2fb5107e48e8bf4cfb8f16ceR57-R64